tensorflow/core/kernels/reverse_sequence_op.cc

```diff
@@ -76,8 +76,8 @@ void CheckErrors(OpKernelContext* context, int batch_dim, int seq_dim) {
   }
 }
 
-template <>
-void CheckErrors<GPUDevice>(OpKernelContext* context, int batch_dim,
+template <typename Tlen>
```

Comment:
I'll remove it. I see Jenkins output, and I'll modify gpu version. What does //tensorflow/core/ops/compat:backwards_compatibility_test do?